server/utils/utils.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package utils
+
+import (
+	"net/url"
+	"path"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func NormalizeInstallURL(mattermostSiteURL, jiraURL string) (string, error) {
```

Comment:
0/5 for what to change it to

```suggestion
func NormalizeInstallURL(mattermostSiteURL, microsoftURL string) (string, error) {
```